src/App.css

```diff
@@ -1,38 +1,109 @@
-.App {
-  text-align: center;
+header h1 {
+  font-size: 50px;
+  margin-left: 10%;
+  color: #f3e3b2;
+  margin-top: 0;
+  margin-bottom: 0;
+  
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+header {
+  background-color: #203769;
+  padding: 0;
+  height: 100px;
+  display:flex;
+  align-items: center;
+}
+
+#navBar {
+  height: 75px;
+  width: auto;
+  display: flex;
+  justify-content: center;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+#navBar button {
+  height: 100%;
+  width: 25vw;
+  font-weight: bolder;
+  font-size: 2vmax;
+  background-color: #D2C89B;
+  color: #203769;
+  border-style: hidden;
+  overflow-wrap: break-word;
+}
+
+#navBar button:hover{
+  cursor: pointer;
+  background-color: #a8a07a;
+}
+
+.dropdown {
+  position: relative;
+  display: inline-block;
+}
+/* Dropdown Content (Hidden by Default) */
+.dropdown-content {
+  display: none;
+  position: absolute;
+  background-color: #D2C89B;
+  min-width: 100%;
+  box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+  z-index: 1;
+}
+/* Links inside the dropdown */
+.dropdown-content a {
+  color:#203769;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: block;
+}
+/* Change color of dropdown links on hover */
+.dropdown-content a:hover {
+  background-color: #a8a07a;
+}
+/* Show the dropdown menu on hover */
+.dropdown:hover .dropdown-content {
+  display: block;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+#reviewList {
   display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  background-color: #203769;
+}
+
+#reviewCard {
+  width: 75vw;
+  max-width: 500px;
+  height: auto;
```

Comment:
This is making some of your cards different heights on larger screen sizes when the grid expands. You may want to give this a fixed value to control this.